config/src/test/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/resource/OAuth2ResourceServerConfigurerTests.java

```diff
@@ -564,7 +667,7 @@ public void requestWhenCustomJwtDecoderExposedAsBeanThenUsed()
 	@Test
 	public void getJwtDecoderWhenConfiguredWithDecoderAndJwkSetUriThenLastOneWins() {
 		OAuth2ResourceServerConfigurer.JwtConfigurer jwtConfigurer =
-				new OAuth2ResourceServerConfigurer().new JwtConfigurer(null);
+				new OAuth2ResourceServerConfigurer(null).jwt();
```

Comment:
We should not pass `null` context.

Reply:
Thank you for catching that! Intended clean up that I missed.